### PR TITLE
Adding python script for creating k8s jobs

### PR DESCRIPTION
**Pull request by sebastianklose**
Creating evaluation jobs from dataset lists (csv) using the kubernetes
python client API.

### CODE REVIEW

File: evaluation/k8s_job_creator/k8s_helper.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+from __future__ import absolute_import
+
+from absl import logging
+from kubernetes import client, config, watch
+
+
+def create_config(in_cluster=False):
+  if in_cluster:
+    logging.info("Loading in cluster config")
```
Comment (pifon2a) [inline evaluation/k8s_job_creator/k8s_helper.py]: nit: in-cluster?

File: evaluation/k8s_job_creator/k8s_job_creator.py [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+            parallelism=1))
+
+    logging.info("Creating evaluation job on kubernetes.")
+    self.try_create_job(job)
+
+  def try_create_job(self, k8s_job):
+    try:
+      response = self.k8s_batch_api.create_namespaced_job(
+          self.namespace, k8s_job, pretty="true")
+      #logging.info(response)
```
Comment (pifon2a) [inline evaluation/k8s_job_creator/k8s_job_creator.py]: remove?

**Review by pifon2a — APPROVED** [review]
Nice, like it a lot. Please, decide, what format of comments do you actually follow (uppercase/lowecase for the first letter).

File: evaluation/k8s_job_creator/k8s_job_creator.py (hunk 2 above)
Comment (sebastianklose) [reply evaluation/k8s_job_creator/k8s_job_creator.py]: Thanks for comments. All addressed!

**Pull request merged**